POS/Integration/06_CreateStore.md

```diff
@@ -56,22 +52,42 @@ The store wizard has been started, see [Start the store wizard](#start-the-store
 
 ![Wizard 01](../../Assets/Screenshots/POS/Management/Stores/StoreWizard/SW01.png "[Store Wizard]")
 
-1. Enter a store name in the *Store Name* field and add a short description if you like.
+1. Enter a store name in the *Store name* field and add a short description if desired.
 
-2. Select a store manager from the list by selecting the corresponding checkbox.
+2. Specify a store manager from the list by selecting the corresponding checkbox.
 
     > [Info] The store managers list displays all the users who are assigned to the POS store manager group. You may select multiple users as store managers.
+
     For detailed information, see [Assign users to the POS groups](./04_AssignUsers.md).
 
 3. Click the [CREATE STORE] button in the bottom right corner of the window.   
-    The *Creating Store...* message is displayed. It may take several minutes until the creation is completed.  
-
-    ![Wizard 01](../../Assets/Screenshots/POS/Management/Stores/StoreWizard/SW01a.png "[Store Wizard]")
-
-    The *Pay desks* wizard window is displayed when the store is created.
-
-
-### Configure the pay desk
+    The *Creating Store...* message is displayed. It may take several minutes until the creation is completed.    
```

Comment:
Prüfe bitte die Einrückung bei den Modulnamen. Ich würde vielleicht die Info hier ein bisschen anders verteilen, damit sie deutlicher wird. Voschlag:
The "Create store" message is displayed. It may take several minutes...

The following entities have been created:
- DataHub
(Tabelle) -> Ich finde Tabellen gut, einfacher zu lesen. Nur weiß ich nicht, ob diese ganze Info hilft oder verwirrt. Grundsätzlich finde ich die Info auch interessant, damit man versteht, was im Hintergrund läuft.
- PIM
(Tabelle)
- OC
- (Tabelle)

The *Pay desks" wizard window is displayed... 
(Ich vermute, dies passiert danach, oder?)